tools/WinObjC.Packaging/WinObjC.Packageable.targets

```diff
@@ -0,0 +1,88 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+
+<!--
+  PACKAGABLE TARGETS
+  Description: The target definitions specific to a WinObjC project intended to be packaged
+  For Use By: A VcxProj that can be packaged into a NuGet
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetOsAndVersion Condition="'$(TargetOsAndVersion)' == ''">Universal Windows</TargetOsAndVersion>
+  </PropertyGroup>
+
+  <!-- Presets for a file's location in the package -->
+  <PropertyGroup>
+    <ShouldAssignPackageContentBasePath Condition="'$(PackageContentBasePath)' == ''">true</ShouldAssignPackageContentBasePath>
+    <HasConfigurationSpecificPackageContents Condition="'$(HasConfigurationSpecificPackageContents)' == ''">false</HasConfigurationSpecificPackageContents>
+    <PackageContentBasePath Condition="'$(ShouldAssignPackageContentBasePath)' == 'true'">build\lib\$(TargetOsAndVersion)\$(PlatformTarget)</PackageContentBasePath>
+    <PackageContentBasePath Condition="'$(ShouldAssignPackageContentBasePath)' == 'true' and '$(HasConfigurationSpecificPackageContents)' == 'true'">$(PackageContentBasePath)\$(Configuration)</PackageContentBasePath>
+
+    <ShouldAssignPackageHeaderBasePath Condition="'$(PackageHeaderBasePath)' == ''">true</ShouldAssignPackageHeaderBasePath>
+    <PackageHeaderBasePath Condition="'$(ShouldAssignPackageHeaderBasePath)' == 'true'">build\include\$(MSBuildProjectName)</PackageHeaderBasePath>
+
+    <ShouldAssignPackageResourcesBasePath Condition="'$(PackageHeaderBasePath)' == ''">true</ShouldAssignPackageResourcesBasePath>
+    <PackageResourcesBasePath Condition="'$(ShouldAssignPackageHeaderBasePath)' == 'true'">build\content\$(MSBuildProjectName)</PackageResourcesBasePath>
+
+    <ShouldAssignPackageBundleBasePath Condition="'$(PackageBundleBasePath)' == ''">true</ShouldAssignPackageBundleBasePath>
+    <PackageBundleBasePath Condition="'$(ShouldAssignPackageBundleBasePath)' == 'true'">build\bundle\$(MSBuildProjectName)</PackageBundleBasePath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GetPackageContentsDependsOn>$(GetPackageContentsDependsOn);$(ComputeLinkInputsTargets);FindPublicHeaders</GetPackageContentsDependsOn>
+  </PropertyGroup>
+
+  <!-- Add MSBuild outputs into the package -->
+  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents"  DependsOnTargets="$(GetPackageContentsDependsOn)" Condition="'$(PresetPackageContents)' == 'true'">
+    <!-- Framework / Static Library Projects -->
+    <ItemGroup Condition="'$(ConfigurationType)' == 'StaticLibrary'">
+      <!-- .lib Files -->
+      <_ImportLibraryFileNames Include="$(OutDir)*.lib" />
```

Comment:
Oh I see a little more what you are doing here but its still wrong. You have this under Condition="'$(ConfigurationType)' == 'StaticLibrary'"

but a static lib doesn't have an import lib. An import lib is paired with a dll so you want this bit to go below and to copy the BuiltProjectOutputGroupOutput to this block as the static lib is the main project output for a static library config.

---
In reply to: [100939160](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100939160) [](ancestors = 100939160)